<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualisationGraphForm</class>
 <widget class="QWidget" name="VisualisationGraphForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1265</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="QGroupBox" name="group_forGView">
       <property name="title">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="group_forElements">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
        <item>
         <widget class="QGroupBox" name="group_info">
          <property name="title">
           <string>Info</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTextEdit" name="text_GView_info"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="group_controls">
          <property name="title">
           <string>Controls</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="button_GView_add">
               <property name="text">
                <string>Дадаць</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_GView_remove">
               <property name="text">
                <string>Выдаліць</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="bottomMargin">
              <number>12</number>
             </property>
             <item>
              <widget class="QPushButton" name="button_AddEdge">
               <property name="text">
                <string>Стварыць сувязь</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_removeEdge">
               <property name="text">
                <string>Адсячы сувязь</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="button_GView_zoomin">
               <property name="text">
                <string>Прыблізіць</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_GView_zoomout">
               <property name="text">
                <string>Аддаліць</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
